documentation/faq/80-integrations.md

```diff
@@ -13,3 +13,21 @@ Put the code to query your database in [endpoints](../docs#routing-endpoints) -
 ### How do I use Axios?
 
 You probably don't need it. We'd generally recommend you just use `fetch` instead. If you insist, you're probably better off using the ESM drop-in replacement `redaxios` [until axios utilizes ESM](https://github.com/axios/axios/issues/1879). Finally, if you still want to use Axios itself, try putting it in `optimizeDeps.include`.
+
+### How do I use Sass/SCSS, Less or Stylus?
```

Comment:
`svelte-preprocess` supports a lot more than this, so it feels weird to single out these libraries.

I'd probably say something more like:

```
### How do I use Babel, CoffeeScript, Less, PostCSS / SugarSS, Pug, scss/sass, Stylus, TypeScript, `global` styles, or replace?

SvelteKit provides [https://github.com/sveltejs/svelte-preprocess](svelte-preprocess) by default. For many of these tools you only need to install the corresponding library such as `npm install -D sass`or `npm install -D less`. See the [https://github.com/sveltejs/svelte-preprocess](svelte-preprocess) docs for full details.

Also see [the examples](faq#how-do-i-use-x-with-sveltekit-how-do-i-setup-library-x) of setting up these and similar libraries.
```

Reply:
Done